process-migration-service/pom.xml

```diff
@@ -514,6 +359,185 @@
         </plugins>
     </build>
     <profiles>
+        <profile>
+            <id>wildfly</id>
```

Comment:
Sorry to change this once again, but I realized that for example the failsafe configuration should be outside this profile since we want to run integration tests but with internal cargo configuration. Right now we either run integration tests with community wildfly + process-exec-plugin or we don't run them at all.

I think the best thing would be to stick with the configuration from kie-server-tests-parent here:
https://github.com/kiegroup/droolsjbpm-integration/blob/master/kie-server-parent/kie-server-tests/pom.xml

So I think the best approach would be to define cargo-general config outside of profile. Then in wildfly profile there should be just wildfly-specific configuration. We can even make process-exec-maven-plugin outside of that profile.

Then as with kie-server we can put 2 profiles here. One which will turn off the container (cargo plugin) + microservice (process-exec-maven-plugin), active by default:

https://github.com/kiegroup/droolsjbpm-integration/blob/a852ff6ff6b94f5763045286af4dcbb0009f08ea/kie-server-parent/kie-server-tests/pom.xml#L1262-L1294

And the second profile to turn off integration tests by default:
https://github.com/kiegroup/droolsjbpm-integration/blob/a852ff6ff6b94f5763045286af4dcbb0009f08ea/kie-server-parent/kie-server-tests/pom.xml#L1331-L1350

So when we run tests internally, it will be possible to just do `-Pcustom-container` and just cargo + microservice setup will be turned off, but failsafe will be active. Moreover, when we do just `clean install`, no tests will be run. And when we do `-Pwildfly`, community KIE Server + microservice will be run and tested.